src/js/rx/reducers/modal.js

```diff
@@ -15,7 +15,7 @@ export default function modal(state = initialState, action) {
       return {
         refill: {
           visible: true,
-          content: null
+          content: action.content
```

Comment:
`content` might not be the right name for it. It makes me expect a string or DOM node. Since you're actually passing along an object of prescription attributes, maybe something like `prescription` or `rx` would be more appropriate.